samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -354,7 +355,14 @@ object SamzaContainer extends Logging {
 
     val chooserFactory = Util.getObj[MessageChooserFactory](chooserFactoryClassName)
 
-    val chooser = DefaultChooser(inputStreamMetadata, chooserFactory, config, samzaContainerMetrics.registry)
+    val sspToTaskCount: mutable.Map[SystemStreamPartition, Int] = mutable.Map[SystemStreamPartition, Int]()
```

Comment:
The explanation was done at multiple places, including BootstrappingChooser, DefaultChooser constructors. Hence didn't add it here. Done. Added a comment, describing the intention.